src/pages/profile-page/profile-page.js

```diff
@@ -0,0 +1,312 @@
+                      ? user.address.country
+                        ? user.address.country
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].region}
+                  name='region'
+                  value={
+                    user.address
+                      ? user.address.region
+                        ? user.address.region
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].city}
+                  name='city'
+                  value={
+                    user.address
+                      ? user.address.city
+                        ? user.address.city
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].street}
+                  name='street'
+                  value={
+                    user.address
+                      ? user.address.street
+                        ? user.address.street
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].buildingNumber}
+                  name='buildingNumber'
+                  value={
+                    user.address
+                      ? user.address.buildingNumber
+                        ? user.address.buildingNumber
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].appartment}
+                  name='appartment'
+                  value={
+                    user.address
+                      ? user.address.appartment
+                        ? user.address.appartment
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].zipcode}
+                  name='zipcode'
+                  value={
+                    user.address
+                      ? user.address.zipcode
```

Comment:
Why do you have `user.address.zipcode` twice here?

Reply:
removed